plugin/ale.vim

```diff
@@ -155,6 +155,9 @@ let g:ale_detail_to_floating_preview = get(g:, 'ale_detail_to_floating_preview',
 " bottom-right and bottom-left
 let g:ale_floating_window_border = get(g:, 'ale_floating_window_border', ['|', '-', '+', '+', '+', '+'])
 
+" Max height for floating window in Neovim
+let g:ale_floating_preview_maxheight = get(g:, 'ale_floating_preview_maxheight', 10)
```

Comment:
Let's rename this to `ale_floating_window_max_height`, and add documentation for it like the other options. Add a reference to it from the `window_border` documentation so people can find it easily in the documentation. With the way it's named, it should be the option that appears directly underneath the border option in the help file, as they are all meant to be sorted alphabetically.